.ci/sidetrack/Jenkinsfile

```diff
@@ -40,7 +40,6 @@ pipeline {
     CAMBPM_LOGGER_LOG_LEVEL = 'DEBUG'
   }
   parameters {
-    booleanParam name: 'STANDALONE', defaultValue: true
```

Comment:
We use the `params.STANDALONE` parameter on line 19 here (the `shouldTrigger(String stageType)` method, to determine if a stage should be executed or not. Removing it would break this functionality. I think we can use `params.UPSTREAM_PR` instead.